backend/src/services/GenerateReporter/GenerateReportService.ts

```diff
@@ -135,6 +145,10 @@ export class GenerateReportService {
     return reporterResponse;
   }
 
+  protected requestIsEmptyButValid(request: GenerateReportRequest): boolean {
+    return Object.keys(request).length === 8 && request.metrics.length === 0;
```

Comment:
request === {}

Reply:
In this method, `Object.keys(request).length === 8` is not equal to `request === {}`, the former indicates that request has the same format as the required request, and the latter indicates that request is only an empty object.